lib/adapters/idb/idb-all-docs.js

```diff
@@ -0,0 +1,184 @@
+'use strict';
```

Comment:
I moved the `allDocs()` function into its own file, same as I did before with `bulkDocs()`.